Tests/DataPipeline/DataPipelineInteractionTests.swift

```diff
@@ -5,19 +5,17 @@ import Foundation
 @testable import SharedTests
 import XCTest
 
-class DataPipelineInteractionTests: UnitTest {
+class DataPipelineInteractionTests: IntegrationTest {
```

Comment:
Same as above.

Updated the test to inherit from `IntegrationTest` instead for two reasons: 1) `DeviceInfoStub` is already available in `IntegrationTest`, and 2) since we are testing multiple aspects, such as the attachment of the device token and the emission of events, `IntegrationTest` seemed like the more suitable option for this test.